### ISSUE TITLE
Maxima binary re-detection is triggered by clipboard cut and other operations

### ISSUE THREAD

**Issue by KubaO** [wxMaxima-developers/wxmaxima#1299]
When observing the Debug Messages pane, each time clipboard cut is performed (e.g. by selecting cells and pressing Ctrl-X), it's clear that Maxima binary is being re-detected - the "Maxima not found as ..." debug message appears.

This does not happen in 19.05.7, but does happen in 20.04.0 and onwards.

**Comment by gunterkoenigsmann** [comment]
On my linux computer I cannot reproduce this. 

I hate issues that happen on some machines, but not on others.

**Comment by KubaO** [comment]
Yep. But on Windows it always happens. I’ll dig into it when I get time. It’s low priority but it does make the cutting process take a bit longer.

**Comment by gunterkoenigsmann** [comment]
Actually giving the time schedule it had to fit in until now it looks like we seem to have made a very good job with this release...

**Comment by KubaO** [comment]
This is a wider problem: every time new configuration is created, the location is checked again. Since we create `Configuration` for all sorts of clipboard and export, this requires some solution. It should reproduce on Linux when *Configuration->Copy->Scalable Vector Graphics* is checked.

At least the following routes of entry are possible:

```
Dirstructure::MaximaDefaultLocation
Configuration::Configuration
OutCommon::OutCommon
Svgout::Svgout | Emfout::Emfout
```

It will also be constructed from `PrintOut::SetupData`.

Maxima shouldn't really be re-detected in those cases.

**Comment by KubaO** [comment]
Fixed by 5123bbad
